src/extract_gms.py

```diff
@@ -362,15 +362,23 @@ def gms_extract(filename,states,min_shift,active_space,flag):
     # the total energies of excited states (1-electron basis)
 
     # *********Here, KS excitation energy -> total excitation energy***************
+    homo = active_space[0] - min_shift # HOMO : number ordering starts from 1, not 0.
     nstates = len(states)
     E = MATRIX(nstates,nstates)
     for i in xrange(nstates):
-        h_indx = states[i].from_orbit[0] - min_shift  # index of the hole orbital w.r.t. the lowest included in the active space orbital
-        e_indx = states[i].to_orbit[0]   - min_shift  # --- same, only for the electron orbital
+        #h_indx = states[i].from_orbit[0] - min_shift  # index of the hole orbital w.r.t. the lowest included in the active space orbital
+        #e_indx = states[i].to_orbit[0]   - min_shift  # --- same, only for the electron orbital
+        #print "%i th excitation" %(i)
+        h_indx = states[i].from_orbit[0] + homo - 1
```

Comment:
these 2 lines + the line 365 - is it true? Why do we need the change with respect to the commented (now deleted) case?

Reply:
This change of extract_gms.py is not needed. I'll take it back to previous version. Sorry for the inconvenience.